packages/service-clients/azure-client/api-report/azure-client.api.md

```diff
@@ -21,16 +22,16 @@ import { ScopeType } from '@fluidframework/protocol-definitions';
 // @public
 export class AzureClient {
     constructor(properties: AzureClientProps);
-    createContainer<const TContainerSchema extends ContainerSchema>(containerSchema: TContainerSchema): Promise<{
+    createContainer<const TContainerSchema extends ContainerSchema>(containerSchema: TContainerSchema, compatMode: CompatMode): Promise<{
```

Comment:
Couple questions here -
1. This is a breaking change, right? Are we planning to add a changeset?
1. Did we consider making `compatMode` an optional param and having it default to either `"1"` or `"2"`? I'm wondering if we could avoid breaking changes.

Reply:
I think @ChumpChief's callout was that we want devs to explicitly opt-in to learn about implications? So our API docs better be good :)

Reply:
Yes, I'll add a changeset.

My initial PR had a default for the param but I received feedback from the team that an explicit decision was preferred.